src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -181,14 +179,11 @@ def handler(ex):
     "Storage Queue Data Contributor"
     "Storage Queue Data Reader"
 
-If you want to use the old authentication method and allow querying for the right account key, please use the "--auth-mode" parameter and "key" value.
+If you want to use the old authentication method and allow querying for the right account key, please use the "--auth-mode" parameter and "key" value. ErrorCode: AuthenticationFailed
                 """
-                logger.error(message)
-                return
+                ex.args = (message,)
             if isinstance(ex, t_error) and ex.status_code == 409 and ex.error_code == 'NoPendingCopyOperation':
-                logger.error(ex.args[0])
-                return
-            raise ex
+                pass
```

Comment:
If pass this exception scenario, how can users know the exception details?

Reply:
The details are in `ex.args[0]`.